libraries/radwordsprocessing/formats-and-conversion/word-file-formats/doc/docformatprovider.md

```diff
@@ -36,12 +36,14 @@ To import a DOC or DOT file, you should use the **Import()** method of **DocForm
 #### **[C#] Example 1: Import document from a file**
 
 {{region cs-radwordsprocessing-formats-and-conversion-doc-docformatprovider_0}}
-	            
-	RadFlowDocument document;
+
+	using Telerik.Windows.Documents.Flow.FormatProviders.Doc;
+	using Telerik.Windows.Documents.Flow.Model;
```

Comment:
Do not add usings to the snippet - it won't build when directly pasted. Instead, use full class names.